packages/ariakit/src/combobox/__examples__/combobox-popover-opening-with-at-least-1-character/test.tsx

```diff
@@ -0,0 +1,48 @@
+import { click, getByRole, press, render } from "ariakit-test";
+import { axe } from "jest-axe";
+import Example from ".";
+
+const getCombobox = () => getByRole("combobox");
+const getPopover = () => getByRole("listbox", { hidden: true });
+const getOption = (name: string) => getByRole("option", { name });
+
+test("ally", async () => {
+  const { container } = render(<Example />);
+  expect(await axe(container)).toHaveNoViolations();
+});
+
+describe("combobox is pristine", () => {
```

Comment:
Let's avoid `describe`. [Flat is better than nested](https://peps.python.org/pep-0020/#:~:text=Flat%20is%20better%20than%20nested.).

Instead, we can just put the context in the test name. For example:

```js
test("popover is not shown on click when combobox is pristine", ...
```